cpp/daal/src/algorithms/pca/transform/pca_transform_dense_default_batch_impl.i

```diff
@@ -105,10 +105,6 @@ services::Status TransformKernel<algorithmFPType, method, cpu>::compute(NumericT
     const algorithmFPType * pBasis = basis.get();
 
     size_t numRowsInBlock = _numRowsInBlock;
-    if (numRowsInBlock < 1)
```

Comment:
would we want to replace this with an assertion/check?

Reply:
In this case, _numRowsInBlock is assigned internally in a header file. We do not expect it to be assigned to zero or negatives.